dagrunner/plugin_framework.py

```diff
@@ -239,7 +239,7 @@ def __call__(
 
 
 class Input(NodeAwarePlugin):
-    def __call__(self, filepath, **kwargs):
+    def __call__(self, filepath, node_properties=None, **kwargs):
```

Comment:
minor change, making node properties explicitly an argument to make consistent with other NodeAwarePlugins